Scatt.java

```diff
@@ -27,13 +27,13 @@
      */
     public static void main(String[] args)
     {
-        System.out.println("Please enter the folder name: ");
+        System.out.print("Please enter the folder path: ");
         String dirName = System.console().readLine();
```

Comment:
@KaraBeason I put the prompt back and added instructions about putting a folder with the .sb2 files in our application directory for the current deliverable. Let me know if you think this works and if it's okay to merge or if any other changes are needed. Thanks!!